test/src/module/command/lockTest.c

```diff
@@ -0,0 +1,83 @@
+/***********************************************************************************************************************************
+Test Command Lock Handler
+***********************************************************************************************************************************/
+#include "storage/posix/storage.h"
+
+#include "common/harnessConfig.h"
+#include "common/harnessStorage.h"
+
+/***********************************************************************************************************************************
+Test Run
+***********************************************************************************************************************************/
+static void
+testRun(void)
+{
+    FUNCTION_HARNESS_VOID();
+
+    // Create default storage object for testing
+    Storage *storageTest = storagePosixNewP(TEST_PATH_STR, .write = true);
```

Comment:
Should this be const Storage *const storageTest?

Reply:
I'd rather leave this for now. In general the way we use const for Storage objects is not consistent with how were use const for other objects. This is because Storage was one of the first objects created and never got a refactor (but I have added it to my list).